Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG JAVA_VERSON=21
+
+FROM mcr.microsoft.com/openjdk/jdk:${JAVA_VERSON}-distroless
+
+WORKDIR /app
+
+VOLUME /app/data
+
+ENV CONFIG_FILE=/app/config.json
+
+COPY build/install/Ember-shadow/lib/Ember-*.jar Ember.jar
+
+ENTRYPOINT ["java", "-XshowSettings:vm", "-XX:MinRAMPercentage=20", "-XX:MaxRAMPercentage=95", "-jar", "Ember.jar", "--config", "${CONFIG_FILE}"]
```

Comment:
This should likely use ENV's with defaults so docker-compose / docker up can easily override them